PK SHINY.md

```
@@ -0,0 +1,197 @@
+[PK SHINY](https://yourshinyappurl.com)
+library(shiny)
+library(nlmixr2)
+library(rxode2)
+library(ggplot2)
+
+# =========================
+# MODEL DEFINITIONS
+# =========================
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)